hassio/src/addon-view/hassio-addon-config.ts

```diff
@@ -0,0 +1,163 @@
+import "@polymer/iron-autogrow-textarea/iron-autogrow-textarea";
+import "@material/mwc-button";
+import "@polymer/paper-card/paper-card";
+import {
+  css,
+  CSSResult,
+  customElement,
+  html,
+  LitElement,
+  property,
+  PropertyValues,
+  TemplateResult,
+} from "lit-element";
+
+import { HomeAssistant } from "../../../src/types";
+import {
+  HassioAddonDetails,
+  setHassioAddonOption,
+  HassioAddonSetOptionParams,
+} from "../../../src/data/hassio";
+import { hassioStyle } from "../resources/hassio-style";
+import { haStyle } from "../../../src/resources/styles";
+import { PolymerChangedEvent } from "../../../src/polymer-types";
+import { fireEvent } from "../../../src/common/dom/fire_event";
+
+@customElement("hassio-addon-config")
+class HassioAddonConfig extends LitElement {
+  @property() public hass!: HomeAssistant;
+  @property() public addon!: HassioAddonDetails;
+  @property() protected error?: string;
```

Comment:
`protected`?

Reply:
It was like that in an existing file under /hassio/src before I started, so just reused that.
Changing!